src/python_binding/Makefile.am

```diff
@@ -0,0 +1,52 @@
+AM_CPPFLAGS = \
+ -I$(top_srcdir)/node/src \
+ -I$(top_srcdir)/node/src/univalue/include \
+ -I$(top_srcdir)/node/src/consensus \
+ -I$(top_srcdir)/node/src/script \
+ -I$(top_srcdir)/node/src/support \
+ -I$(top_srcdir)/contrib/secp256k1/include \
+ -I$(top_srcdir)/src/common \
+ -I$(top_srcdir)/src/p2p \
+ -I$(top_srcdir)/src/core \
+ -I$(top_srcdir)/contrib/cex\
+ $(PYTHON_CPPFLAGS)
+
+SWIG_SOURCES_CPP = \
+ $(top_srcdir)/node/src/support/cleanse.cpp \
+ $(top_srcdir)/node/src/bech32.cpp \
+ $(top_srcdir)/node/src/primitives/transaction.cpp
+
+SWIG_FLAGS = -stdstl
+SWIG_SOURCES = l15-core-pybind.i
+SWIG_PYTHON_MODULE = libl15-core-pybind-wrap.cxx
+CLEANFILES = $(SWIG_PYTHON_MODULE) libl15-core-pybind.py
+
+BUILT_SOURCES = $(SWIG_PYTHON_MODULE)
+EXTRA_DIST = $(SWIG_SOURCES)
+
+$(SWIG_PYTHON_MODULE): $(SWIG_SOURCES)
+	$(SWIG) -c++ -python -o $@ $<
+
+AM_LDFLAGS = -L$(top_builddir)/node/src \
+             -L$(top_builddir)/node/src/crypto
+
+lib_LTLIBRARIES = _libl15-core-pybind.la
+
+_libl15_core_pybind_la_SOURCES = $(SWIG_PYTHON_MODULE) $(SWIG_SOURCES_CPP)
+_libl15_core_pybind_la_CXXFLAGS = $(PYTHON_INCLUDES) $(PYTHON_CPPFLAGS) -fPIC
```

Comment:
-fPIC should arrive from common build parameters instead of being hardcoded in-place